presto-product-tests/src/main/java/io/prestosql/tests/TestGroups.java

```diff
@@ -67,6 +67,7 @@
     public static final String KAFKA = "kafka";
     public static final String TWO_HIVES = "two_hives";
     public static final String ICEBERG = "iceberg";
+    public static final String AVRO = "avro";
```

Comment:
i think this should be part of storage_formats group (no need for new group), especially if it exploits new way of interacting between presto, file format, and file system.

Reply:
> i think this should be part of storage_formats group (no need for new group), 

Avro tests are still part of `storage_formats`. This group allows to exclude avro tests until https://github.com/prestosql/presto/issues/3794 is fixed (there is also TODO in `bin/product-tests-suite-5.sh`)

Reply:
Some are. But some are not (`presto-product-tests/src/main/java/io/prestosql/tests/hive/TestAvroSchemaUrl.java`, `presto-product-tests/src/main/java/io/prestosql/tests/hive/TestAvroSchemaEvolution.java`)

maybe those two shouldnt' be storage_formats though

Reply:
oh, the TestAvroSchemaUrl should be. Please add